### ISSUE TITLE
Warning when submitting answers when $permissionLevels{dont_log_past_answers} is set.

### ISSUE THREAD

**Issue by somiaj** [openwebwork/webwork2#1363]
I was testing out some problems as my admin account and when I try to submit an answer to a problem I get an error.

```
WeBWorK Warnings

WeBWorK has encountered warnings while processing your request. If this occured when viewing a problem,
it was likely caused by an error or ambiguity in that problem. Otherwise, it may indicate a problem with the
WeBWorK system itself. If you are a student, report these warnings to your professor to have them corrected.
If you are a professor, please consult the warning output below for more information.

Warning messages

    Use of uninitialized value in concatenation (.) or string at
    /home/webwork/webwork2/lib/WeBWorK/ContentGenerator/ProblemUtil/ProblemUtil.pm line 186.
```

In testing I only get this warning when submitting a answers as a professor or admin. Submitting answers as a ta or lower doesn't give me this warning. Here I'm unsure if the goal is to not allow profs/admins to store answers in the database or not, but in either case I don't think getting a warning should happen.

Also it appears there is a spelling error with `occured` in that output.

**Comment by drgrice1** [comment]
Professors and admins are allowed to submit answers.  How are you getting this warning?  You will need to be more specific as to how to reproduce this.

**Comment by somiaj** [comment]
I get this warning when I submit answers to any question as my admin user. I do get feedback if the answer was correct or not, but it is not saved to the database.

**Comment by drgrice1** [comment]
Is this on WeBWorK 2.16?  I am not seeing this.  If I submit answers as an admin user, there is no error, and the answers are stored in the database.  This is true for all versions of WeBWorK in fact.  No error and answers are stored.

**Comment by somiaj** [comment]
Okay, I think I have tracked down the issue. I went and compared my `localOverrides.conf` to `localOverrides.conf.dist` and noticed that I uncommitted the following line: `$permissionLevels{dont_log_past_answers} = "professor";`

If I go and comment out that line, I no longer get the warning. Maybe I misunderstood what that did, I was thinking it would log past answers in the past answer database (since I do lots of testing figured I didn't need to log my past answers), but thought it would at least keep the most current `last_answer`.
